server/src/main/java/org/opensearch/action/admin/indices/stats/IndicesStatsRequest.java

```diff
@@ -282,6 +282,15 @@ public IndicesStatsRequest includeUnloadedSegments(boolean includeUnloadedSegmen
         return this;
     }
 
+    public IndicesStatsRequest segmentReplication(boolean segmentReplication) {
```

Comment:
Given RemoteSegmentStats is inside of SegmentsStats, I think it would make sense for us to place ReplicationStats inside of SegmentsStats as well?